hil-test/tests/parl_io.rs

```diff
@@ -169,21 +178,28 @@ mod tests {
         let clock_out_pin = ClkOutPin::new(clock_tx);
         let clock_in_pin = RxClkInPin::new(clock_rx, SampleEdge::Normal);
 
-        let pio = ParlIoFullDuplex::new(ctx.parl_io, ctx.dma_channel, Rate::from_mhz(40)).unwrap();
+        let pio = ParlIoFullDuplex::new(ctx.parl_io, ctx.dma_channel).unwrap();
 
         let pio_tx = pio
             .tx
             .with_config(
                 tx_pins,
                 clock_out_pin,
-                0,
-                SampleEdge::Invert,
-                BitPackOrder::Lsb,
+                TxConfig::default()
+                    .with_frequency(Rate::from_mhz(40))
+                    .with_sample_edge(SampleEdge::Invert)
+                    .with_bit_order(BitPackOrder::Lsb),
             )
             .unwrap();
         let pio_rx = pio
             .rx
-            .with_config(rx_pins, clock_in_pin, BitPackOrder::Lsb, None)
+            .with_config(
+                rx_pins,
+                clock_in_pin,
+                RxConfig::default()
+                    .with_frequency(Rate::from_mhz(40))
```

Comment:
Are these settings independent in TX and RX?

Reply:
Indeed! Two separate set of registers